### PR TITLE
httplib2: initial integration.

### PR DESCRIPTION
**Pull request by DavidKorczynski**

### CODE REVIEW

File: projects/httplib2/fuzz_request.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# limitations under the License.
+
+import sys
+import httplib2
+import atheris
+import tests
+
+def TestOneInput(input_bytes):
+    http = httplib2.Http()
+    ib = b"HTTP/1.0 200 OK\r\n" + input_bytes
```
Comment (temoto) [inline projects/httplib2/fuzz_request.py:23]: I don't know what is the good way here. This code will generate invalid http message almost all the time.

Proper semi-correct http message generator is a lot of work, maybe someone did it before?

File: projects/httplib2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN git clone --depth 1 https://github.com/httplib2/httplib2/
```
Comment (temoto) [inline projects/httplib2/Dockerfile:18]:
```suggestion
RUN git clone --depth 1 https://github.com/httplib2/httplib2
```

**Comment by temoto** [comment]
maybe @cdent wants to receive notifications too

**Comment by DavidKorczynski** [comment]
@inferno-chromium @jonathanmetzman @oliverchang Fuzzers are now merged upstream and this one is ready to go.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**